Cinema system/Cinema system/Models/Utils/Response.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System.Collections.Generic;
+
+namespace CinemaSystem.Models
+{
+    public class Response
+    {
+        public bool IsSuccessful { get; set; }
+        public List<string> Description { get; set; }
```

Comment:
Properties of enumerable types should have names in plural form.
Also it's not quite clear what this description is for. Is it response description? Or some other?